scala-ws-client/src/test/scala/com/m3/octoparts/ws/RichPartRequestSpec.scala

```diff
@@ -0,0 +1,29 @@
+package com.m3.octoparts.ws
+
+import com.m3.octoparts.model.{ PartRequestParam, PartRequest }
+import org.scalatest._
+import play.api.http.Writeable
+
+class RichPartRequestSpec extends FunSpec with Matchers {
+
+  describe("#withBody") {
+
+    import PartRequestEnrichment._
+    implicit val tuple3IntWriteable = new Writeable[(Int, Int, Int)](transform = { a => s"${a._1 + a._2 + a._3}".getBytes }, None)
```

Comment:
Good point; I've changed this to use UTF-8, but would like to discuss whether we should just leave this as the default charset and change the String building to also use the default charset.

After all, when it gets turned into JSON, it will be converted into UTF-8 anyway.